internal/util/errors.go

```diff
@@ -33,6 +33,9 @@ var (
 	ErrSnapNameConflict = errors.New("snapshot name conflict")
 	// ErrPoolNotFound is returned when pool is not found.
 	ErrPoolNotFound = errors.New("pool not found")
+
+	// ErrIncompleteDataInOMAP is returned when image details are not found in pool.
```

Comment:
When image details are not found in OMAP....Isnt it ?

Reply:
Yes correct. updated it now.